src/components/ZclEndpointCard.vue

```diff
@@ -317,7 +293,7 @@ export default {
               }
             }
           })
-          this.selectedservers = [...enabledServers, ...enabledServers]
+          this.selectedservers = [...enabledServers, ...enabledClients]
```

Comment:
why did it change to enabledClients?

Reply:
Here we have to enableCluster but we should use enableClient for one of them I change that to fix issue